website/src/components/lifeCycle/index.js

```diff
@@ -22,8 +22,10 @@ export default function Lifecycle(props) {
   if (!props.status || (Array.isArray(props.status) && props.status.length === 0)) {
     return null;
   }
-  // Check if props.status is an array or a single value
-  const statuses = Array.isArray(props.status) ? props.status : [props.status];
+  // Check if props.status is an array or a single value amd to handle strings separated by commas
+  const statuses = typeof props.status ==='string'
+  ?props.status.split(',').map(s => s.trim()) 
+  : Array.isArray(props.status) ? props.status : [props.status];
```

Comment:
Now that we're always passing in a string, there's a couple adjustments we can make:

You're on the right track with using `.split(',')`! I recommend updating this section and simply setting `const statuses = props?.status?.split(',')`. This will always create an array, even if one item is passed in without a comma.

For example, passing in `status="enterprise"` will set `props?.status?.split(',')` to `['enterprise']`. If the `status` prop isn't set on a component, `statuses` will become undefined.

I then recommend moving this line above the `if` check on line 22. We can then update the if check to: `if(!statuses?.length)`. This will check if `statuses` has a value and is an array. If not, return null. Otherwise apply the html.

Let me know if you have any questions!

Reply:
ok awesome! thank you so much @JKarlavige ! i made those changes. one thing i kept though was 
```
  if (!props.status || (Array.isArray(props.status) && props.status.length === 0)) {
    return null;
  }```

because if i didn't add a status, the header would display a dash:
![Screenshot 2024-02-14 at 18 29 14](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/6146fe18-c3e9-4dab-a084-059c7f803216)

Reply:
Nice catch, was able to recreate when passing an empty string into the status prop: `status=""`. I merged your if check into the updated one so we only have 1 if check. The `!props.status` is enough to verify if `status` is an empty string.

Added the component to a few headers to double-check the build one more time, if the build passes i'll get this merged!